daycare/forms.py

```diff
@@ -0,0 +1,31 @@
+from django import forms
+from .models import DayCare, Area
+from django.db.models import Max, Min
+
+
+class DayCareSearchForm(forms.Form):
+    min_price = DayCare.objects.all().aggregate(Min('price_per_day')).get('price_per_day__min')
+    max_price = DayCare.objects.all().aggregate(Max('price_per_day')).get('price_per_day__max')
```

Comment:
I'm not sure the user should be aware of the min\max values a daycare can use for his pricing

Reply:
I guess you right .

Reply:
Fixed